tests/php/Forms/TreeDropdownFieldTest.php

```diff
@@ -363,4 +363,47 @@ public function testTreeBaseID()
             'TreeBaseId is included in the default schema data'
         );
     }
+
+    public function testSearchFilter()
+    {
+        $field = new TreeDropdownField('TestTree', 'Test tree', TestObject::class);
+
+        // case-insensitive search against keyword 'zero' for pages
+        $request = new HTTPRequest('GET', 'url', ['search'=>'zero', 'format' => 'json', 'flatList' => '1' ]);
```

Comment:
Needs a bit of linting throughout the test
```suggestion
        $request = new HTTPRequest('GET', 'url', ['search' => 'zero', 'format' => 'json', 'flatList' => '1' ]);
```

Reply:
fixed